vizro-core/changelog.d/20231001_000340_huong_li_nguyen_fix_test_interdependence.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category.
+
+-->
+<!--
+### Changed
+
+- A bullet item for the Changed category.
+
+-->
+<!--
+### Deprecated
+
+- A bullet item for the Deprecated category.
+
+-->
+
+### Fixed
+
+- Fix unit test interdependence issue due to shared dash.page_registry ([#84](https://github.com/mckinsey/vizro/pull/84))
```

Comment:
Overall I think we should not try to be ironclad about any rule we come up with. If it's very important I would mention it either way. Most importantly the changelog is for users and contributors alike to understand what changed between versions.

In general I think it is more important for users though, as they are more likely to compare versions and decide which one to use. So would always be more in favour of keeping keeping the focus on issues that concern/affect the user.

So this one is probably a 50/50, I personally would not really care to read it in a changelog, but I can see a case where this could be relevant. But most of the time it feels like clogging.

Reply:
Fully agree! I would leave this in then